nautobot/extras/api/views.py

```diff
@@ -969,13 +970,14 @@ def dry_run(self, request, pk):
             raise PermissionDenied("You do not have permission to run this job.")
 
         # Immediately enqueue the job
-        job_kwargs = job_model.job_class.prepare_job_kwargs(scheduled_job.kwargs.get("data", {}))
+        job_class = get_job(job_model.class_path, reload=True)
+        job_kwargs = job_class.prepare_job_kwargs(scheduled_job.kwargs.get("data", {}))
```

Comment:
Unrelated to this PR but this is a bug and should be:

```suggestion
        job_kwargs = job_class.prepare_job_kwargs(scheduled_job.kwargs or {})
```

Reply:
Good catch! Surprised our tests don't pick up on this - I'll fix it and look at adding a quick test case.